python/generational/decisions.py

```diff
@@ -36,7 +36,8 @@
     "mating": 2     # need energy to mate, as well as attraction
 }
 
-class Work():
+
```

Comment:
acroding to PEP, two line breaks between classes

Reply:
thanks for the information 👍  finally, a python standard I like 😄 🎉